data/human/free worlds side plots.txt

```diff
@@ -11,6 +11,88 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+
+
+mission "FW Convoy to Zug"
+	name "Convoy to Zug"
+	description "Escort these Freighters to Zug for an important project. Expect pirate raids."
+	source
+		government "Free Worlds"
+		near Zubeneschamali 2 4
+	destination "Zug"
+	to offer
+		has "event: war begins"
+		has "FW Syndicate Diplomacy 1D: done"
+		not "fw bloodsea done"
```

Comment:
I think it'd be more appropriate to start offering this after the basic shipyard is already in place, then stop offering it once the player gets started with the middle chapter, as the Dreadnought is mentioned shortly into the middle.
```suggestion
		has "event: tarazed diplomacy"
		not "FW Diplomacy 1: done"
```